packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * External dependencies
+ */
+import removeAccents from 'remove-accents';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHeading as Heading,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState, useMemo, useCallback } from '@wordpress/element';
+import { useEntityRecords } from '@wordpress/core-data';
+import { decodeEntities } from '@wordpress/html-entities';
+
+/**
+ * Internal dependencies
+ */
+import Page from '../page';
+import Link from '../routes/link';
+import AddedBy from '../list/added-by';
+import { TEMPLATE_POST_TYPE } from '../../utils/constants';
+import { DataViews } from '../dataviews';
+import {
+	useResetTemplateAction,
+	useTrashTemplateAction,
+} from './template-actions';
+
+const EMPTY_ARRAY = [];
+const defaultConfigPerViewType = {
+	list: {},
+	grid: {},
+};
+
+const DEFAULT_VIEW = {
+	type: 'list',
+	search: '',
+	page: 1,
+	perPage: 5, //20,
```

Comment:
Can we make this an even number, so it works well with grid? https://github.com/WordPress/gutenberg/pull/55906 went for 20.

Reply:
That's for my testing right now with pages. I have less than 20 and it's easier for me to have a smaller number.